intermediate_notebooks/E2E/taxi/NYCTaxi-E2E.ipynb

```diff
@@ -106,113 +153,208 @@
+       "      <td>40.781330</td>\n",
+       "      <td>1</td>\n",
+       "      <td>-73.977264</td>\n",
+       "      <td>40.758514</td>\n",
+       "      <td>8.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  pickup_datetime dropoff_datetime  passenger_count  trip_distance  \\\n",
+       "0      2016-01-01       2016-01-01                2           1.10   \n",
+       "1      2016-01-01       2016-01-01                5           4.90   \n",
+       "2      2016-01-01       2016-01-01                1          10.54   \n",
+       "3      2016-01-01       2016-01-01                1           4.75   \n",
+       "4      2016-01-01       2016-01-01                3           1.76   \n",
+       "\n",
+       "   pickup_longitude  pickup_latitude  rate_code  dropoff_longitude  \\\n",
+       "0        -73.990372        40.734695          1         -73.981842   \n",
+       "1        -73.980782        40.729912          1         -73.944473   \n",
+       "2        -73.984550        40.679565          1         -73.950272   \n",
+       "3        -73.993469        40.718990          1         -73.962242   \n",
+       "4        -73.960625        40.781330          1         -73.977264   \n",
+       "\n",
+       "   dropoff_latitude  fare_amount  \n",
+       "0         40.732407          7.5  \n",
+       "1         40.716679         18.0  \n",
+       "2         40.788925         33.0  \n",
+       "3         40.657333         16.5  \n",
+       "4         40.758514          8.0  "
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "## Handling 2016's Mid-Year Schema Change\n",
+    "base_path = \"/home/notebooks/extended/dont_rm_data_taxi/taxi/\"\n",
+    "months = [str(x).rjust(2, '0') for x in range(1, 3)]\n",
+    "valid_files = [base_path+'yellow_tripdata_2016-'+month+'.csv' for month in months]\n",
     "\n",
-    "In 2016, only January - June CSVs have the columns we need. If we try to read `base_path+2016/yellow_*.csv`, Dask will not appreciate having differing schemas in the same DataFrame.\n",
     "\n",
-    "Instead, we'll need to create a list of the valid months and read them independently."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "months = [str(x).rjust(2, '0') for x in range(1, 7)]\n",
-    "valid_files = [base_path+'2016/yellow_tripdata_2016-'+month+'.csv' for month in months]\n",
-    "valid_files"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    "df_2016_list = []\n",
     "# read & clean 2016 data and concat all DFs\n",
-    "df_2016 = dask_cudf.read_csv(valid_files).map_partitions(clean, remap, must_haves)\n",
+    "\n",
+    "for file in valid_files:\n",
```

Comment:
Why the change here?

Reply:
I was reading from local files, so I had to change it; reverted it back now